app/src/main/res/layout/group_making_settings.xml

```diff
@@ -60,7 +60,7 @@
         <EditText
             android:id="@+id/num_of_groups"
             android:text="@string/default_number_of_groups"
-            style="@style/two_number_setting"
+            style="@style/groups_number_setting"
```

Comment:
Why don't we apply the new style to the other EditText as well?

Reply:
I didn't think the numberOfNamesPerGroup may be more than 99 names per group